### PR TITLE
Add Code of conduct information

### PR DESCRIPTION
**Pull request by povilasv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ Quick links:
 The Prometheus & The Ecosystem community meeting is intended to provide a holistic overview of community activities, critical release information, and governance updates. 
 It also provides a forum for discussion of project-level concerns that might need a wider audience.
 
+## Code Of Conduct
+
+The meeting follows [Prometheus Code of Conduct](https://github.com/prometheus/prometheus/blob/master/code-of-conduct.md). If you notice a violation of the Code of Conduct, reach out to Mishi Choudhary at mishi@linux.com.
```
Comment (beorn7) [inline README.md]: I'd say “…follows the Prometheus Code of Conduct…”.

And perhaps use a mailto link? [Mishi Choudhary](mailto:mishi@linux.com)

File: README.md [hunk 2]
```diff
@@ -17,6 +17,10 @@ Quick links:
 The Prometheus & The Ecosystem community meeting is intended to provide a holistic overview of community activities, critical release information, and governance updates. 
 It also provides a forum for discussion of project-level concerns that might need a wider audience.
 
+## Code Of Conduct
+
+The meeting follows the [Prometheus Code of Conduct](https://github.com/prometheus/prometheus/blob/master/code-of-conduct.md). If you notice a violation of the Code of Conduct, reach out to Mishi Choudhary at [mishi@linux.com])(mailto:mishi@linux.com).
```
Comment (beorn7) [inline README.md]: There is now a spurious closing ')'. If you want to explicitly list the mail address, you don't even need the mailto link. Github will render an email address as a mailto link anyway. My idea was to write
```[Mishi Choudhary](mishi@linux.com)```

But that would mean that the mail address itself is not visible as such. I guess a matter of preference. I don't feel strongly. Just wanted to point out the option.

Comment (povilasv) [reply README.md]: I did it with mailto link and visible email. Tested with chrome, automatically opens gmail tab :)

Comment (beorn7) [reply README.md]: What I tried to say is that you don't even need the mailto: link if you opt for having the mail address visible. The GH markdown renderer will add it automatically. But it doesn't hurt to put it there explicitly, either.

**Pull request merged**